client/app/components/home/home.spec.js

```diff
@@ -3,6 +3,7 @@ import HomeModule from "./home";
 describe("Home", () => {
   let $rootScope, $componentController;
 
+  beforeEach(window.module("Authentication")); //to load hookProvider
```

Comment:
in other places it imports the angular module and passes the value into the `module()` function (it is just the string `"Authentication"`, but would keep it consistent)